lib/xcodeproj/project/object/native_target.rb

```diff
@@ -308,7 +308,7 @@ def new_shell_script_build_phase(name = nil)
         # @return [void]
```

Comment:
The documented return type was `void`, so nobody _should_ have been expecting a stable return value before. So this is not a breaking change 👍